site_scons/prereq_tools/base.py

```diff
@@ -696,6 +696,8 @@ def __init__(self, env, variables, config_file=None, arch=None):
                        'Specifies name of pkg-config to load for MPI', None))
         self.add_opts(BoolVariable('FIRMWARE_MGMT',
                                    'Build in device firmware management.', 0))
+        self.add_opts(BoolVariable('STACK_MMAP',
+                                   'Allocate ABT ULTs stacks with mmap()', 1))
```

Comment:
Just so I understand this code, it's a build option, but it's off by default for all build types?  Assuming it's stable then wouldn't we want it on for debug builds?